project/plugins.sbt

```diff
@@ -1,3 +1,5 @@
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.5")
 
 addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.9")
+
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2")
```

Comment:
We can replace this line with just `addDependencyTreePlugin` e.g. https://github.com/guardian/mobile-apps-api/blob/main/project/plugins.sbt#L15